frigate/util.py

```diff
@@ -926,6 +926,11 @@ def get_nvidia_gpu_stats() -> dict[str, str]:
         "--format=csv",
     ]
 
+    if ("CUDA_VISIBLE_DEVICES" in os.environ):
```

Comment:
You're going to need to handle the case where the value for this env variable is `ALL` in which case we should not append an --id (same for the other env check as well)

Reply:
Probably just only append it if it's an int